src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="PreferAsSpanOverSubstringDescription" xml:space="preserve">
+    <value>'AsSpan' accomplishes the same thing as 'Substring' without incurring the allocation and O(n) string copy.</value>
+  </data>
+  <data name="PreferAsSpanOverSubstringMessage" xml:space="preserve">
+    <value>Prefer 'AsSpan' over 'Substring' when span-based overloads are available.</value>
```

Comment:
Looks like the resx analyzer is not working. Message is not supposed to end with period.

Reply:
Fixed.